src/pybind/mgr/orchestrator.py

```diff
@@ -716,6 +718,69 @@ def __init__(self):
         # some replicaset special sauce for autoscaling?
         self.extended = {}
 
+        # Object with specific settings for the service
+        self.service_spec = None
+
+class RGWSpec(object):
+    """
+    Settings to configure a multisite Ceph RGW
+    """
+    def __init__(self, hosts=None, rgw_multisite=True, rgw_zone="Default_Zone",
```

Comment:
hm. RGW already default to sane values. Can we leverage the default values of RGW instead of defining new ones here? 

e.g. setting everything to `None` and then omit them when generating the config?

Reply:
Yep no problem with that.
 What are the default sane values? (i havent' found this default values ... is Rook using them?)